plugins/importtext_procedure.cc

```diff
@@ -1039,113 +881,83 @@ struct ImportTextProcedureWorkInstance
 
                 const size_t numInputColumn = inputColumnNames.size();
 
-	        const char * errorMsg = parseFixedWidthCsvRow(line, length, &values[0],
-                                                              numInputColumn,
-                                                              separator, quote, encoding,
-                                                              replaceInvalidCharactersWith,
-                                                              isTextLine,
-                                                              hasQuoteChar);
-
+	        const char * errorMsg
+                    = parseFixedWidthCsvRow(line, length, &values[0],
+                                            numInputColumn,
+                                            separator, quote, encoding,
+                                            replaceInvalidCharactersWith,
+                                            isTextLine,
+                                            hasQuoteChar);
+                
                 if (errorMsg)
-	            return handleError(errorMsg, actualLineNum, line - lineStart + 1, string(line, length));
+	            return handleError(errorMsg, actualLineNum,
+                                       line - lineStart + 1,
+                                       string(line, length));
 
-	        //cerr << "got values " << jsonEncode(vector<CellValue>(values, values + inputColumnNames.size())) << endl;
-	                
-	        auto row = scope.bindRow(&values[0], ts, actualLineNum, 0 /* todo: chunk ofs */);
+	        auto row = scope.bindRow(&values[0], ts, actualLineNum,
+                                         0 /* todo: chunk ofs */);
 
 	        // If it doesn't match the where, don't add it 
 	        if (!isWhereTrue) {
 	            ExpressionValue storage;
-	            if (!whereBound(row, storage, GET_LATEST).isTrue())
+	            if (!whereBound(row, storage, GET_ALL).isTrue())
```

Comment:
We don't require GET_ALL.  However, in the context, GET_ALL is far, far faster than GET_LATEST, and it's not possible for a row to have more than one timestamp.  I have difficulty imagining how this could cause a failure.